rosdep/base.yaml

```
@@ -1319,6 +1319,13 @@ gstreamer1.0-alsa:
   fedora: [gstreamer1-plugins-base]
   rhel: [gstreamer1-plugins-base]
   ubuntu: [gstreamer1.0-alsa]
+gstreamer1.0-gl:
+  arch: [gst-plugins-base]
+  debian: [gstreamer1.0-gl]
```

I think we should add an entry for `stretch`, which is still a supported LTS but does not have `gstreamer1.0-gl`:
```suggestion
  debian:
    '*': [gstreamer1.0-gl]
    stretch: null
```